src/stylesheets/core/_properties.scss

```diff
@@ -313,6 +314,36 @@ $uswds-properties: (
       'mono-4':	   lh(mono, 4),
       'mono-5':	   lh(mono, 5),
       'mono-6':	   lh(mono, 6),
+      'cond-1':	   lh(cond, 1),
+      'cond-2':	   lh(cond, 2),
+      'cond-3':	   lh(cond, 3),
+      'cond-4':	   lh(cond, 4),
+      'cond-5':	   lh(cond, 5),
+      'cond-6':	   lh(cond, 6),
+      'heading-1': lh(heading, 1),
```

Comment:
Please align values.

Reply:
oh huh, I see it aligned in my editor... maybe there's a hidden tab or something